Antonio Bonet | html-exercises

tonyblu331 opened the pull request.

tonyblu331 committed:
Created Antonio.html

tonyblu331 committed:
Section 1 (Tooltip, IMG Size, Href, and IMG Alt)

Added Tooltip, fixed Image width, and height fixed <a href> and added Img alt.

tonyblu331 committed:
HTML Paragraphs

tonyblu331 committed:
HTML Headings and Styles

tonyblu331 committed:
HTML Formatting

tonyblu331 committed:
HTML Quotations

tonyblu331 committed:
HTML Links

tonyblu331 committed:
HTML Images

tonyblu331 committed:
HTML Tables

tonyblu331 committed:
HTML Links

tonyblu331 committed:
HTML iFrame

tonyblu331 committed:
HTML Forms

tonyblu331 committed:
HTML Input Types

iamcucusa commented:
Hello, you still have some error, please give it another try and validate your code. See the following file to see your results. 
[Archive.zip](https://github.com/iamcucusa/html-exercises/files/7506014/Archive.zip)

tonyblu331 committed:
Validated using W3C

There is a warning of using the H1 as a top-level element but this is part of the HTML headings sections, so it can't really be fixed unless the exercises itself its changed.
